lib/recurly/hosted-field.js

```diff
@@ -85,21 +87,21 @@ export class HostedField extends Emitter {
   }
 
   inject () {
-    this.target.innerHTML = `
-      <div class="${this.classList}">
-        <iframe
-          src="${this.url}"
-          border="0"
-          frameborder="0"
-          allowtransparency="true"
-          scrolling="no">
-        </iframe>
-      </div>
-    `;
-
-    this.container = this.target.children[0];
-    this.iframe = this.container.children[0];
-    this.window = this.iframe.contentWindow;
+    this.target.innerHTML = dom.hostedFieldHtml(this.classList, this.url);
+
+    if (dom.isMobile()) {
+      this.container = this.target.children[0]; // recurly-hosted-field
+      this.tabbingProxy = this.container.children[0];
+      this.iframe = this.container.children[1]; // iframe
+      this.window = this.iframe.contentWindow;
+      this.tabbingProxy.addEventListener('focus', () => {
+        this.focus();
+      });
```

Comment:
Since this method is more concerned with injecting into the DOM, this should move out to its own method akin to `bindDeferredFocus`